### PR TITLE
refactor: move component into a single package

### PR DESCRIPTION
**Pull request by himself65**
preview: https://test-affine-ui.vercel.app/

### CODE REVIEW

**Comment by himself65** [comment]
There're some incompatibilities after moving to a single package. Trying to figure it out

File: packages/app/src/components/workspace-setting/SyncPage.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ export const SyncPage = ({ workspace }: { workspace: WorkspaceUnit }) => {
           <Content weight={500}>{t('is a Local Workspace')}</Content>
         </FlexWrapper>
         <p>{t('Local Workspace Description')}</p>
-        <Wrapper marginTop="32px">
+        <Wrapper margin="32px 0 0 0">
```
Comment (himself65) [inline packages/app/src/components/workspace-setting/SyncPage.tsx]:
```suggestion
        <Wrapper marginTop="32px">
```

File: packages/app/src/components/workspace-setting/member/MembersPage.tsx [hunk 2]
```diff
@@ -171,7 +171,7 @@ export const MembersPage = ({ workspace }: { workspace: WorkspaceUnit }) => {
         fontSize: '18px',
       }}
     >
-      <Wrapper marginBottom="32px">{t('Collaboration Description')}</Wrapper>
+      <Wrapper margin="0 0 32px 0">{t('Collaboration Description')}</Wrapper>
```
Comment (himself65) [inline packages/app/src/components/workspace-setting/member/MembersPage.tsx]:
```suggestion
      <Wrapper marginBottom="32px">{t('Collaboration Description')}</Wrapper>
```

File: packages/app/src/components/workspace-setting/ExportPage.tsx [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-import { Button } from '@/ui/button';
+import { Wrapper } from '@affine/component';
+import { Button } from '@affine/component';
 export const ExportPage = ({ workspace }: { workspace: WorkspaceUnit }) => {
   const { t } = useTranslation();
   console.log(workspace);
   return (
     <>
-      <Wrapper marginBottom="32px"> {t('Export Description')}</Wrapper>
+      <Wrapper margin="0 0 32px 0"> {t('Export Description')}</Wrapper>
```
Comment (himself65) [inline packages/app/src/components/workspace-setting/ExportPage.tsx]:
```suggestion
      <Wrapper marginBottom="32px"> {t('Export Description')}</Wrapper>
```

File: packages/app/src/components/workspace-setting/PublishPage.tsx [hunk 4]
```diff
@@ -30,9 +30,9 @@ export const PublishPage = ({ workspace }: { workspace: WorkspaceUnit }) => {
     if (workspace.published) {
       return (
         <>
-          <Wrapper marginBottom="32px">{t('Published Description')}</Wrapper>
+          <Wrapper margin="0 0 32px 0">{t('Published Description')}</Wrapper>
```
Comment (himself65) [inline packages/app/src/components/workspace-setting/PublishPage.tsx]:
```suggestion
          <Wrapper marginBottom="32px">{t('Published Description')}</Wrapper>
```

File: packages/app/src/components/workspace-setting/PublishPage.tsx [hunk 5]
```diff
@@ -30,9 +30,9 @@ export const PublishPage = ({ workspace }: { workspace: WorkspaceUnit }) => {
...
 
-          <Wrapper marginBottom="12px">
+          <Wrapper margin="0 0 32px 0">
```
Comment (himself65) [inline packages/app/src/components/workspace-setting/PublishPage.tsx]:
```suggestion
          <Wrapper marginBottom="12px">
```

File: packages/app/src/components/workspace-setting/PublishPage.tsx [hunk 6]
```diff
@@ -64,7 +64,7 @@ export const PublishPage = ({ workspace }: { workspace: WorkspaceUnit }) => {
 
     return (
       <>
-        <Wrapper marginBottom="32px">{t('Publishing Description')}</Wrapper>
+        <Wrapper margin="0 0 32px 0">{t('Publishing Description')}</Wrapper>
```
Comment (himself65) [inline packages/app/src/components/workspace-setting/PublishPage.tsx]:
```suggestion
        <Wrapper marginBottom="32px">{t('Publishing Description')}</Wrapper>
```

File: packages/app/src/components/workspace-setting/PublishPage.tsx [hunk 7]
```diff
@@ -82,7 +82,7 @@ export const PublishPage = ({ workspace }: { workspace: WorkspaceUnit }) => {
 
   return (
     <>
-      <Wrapper marginBottom="32px">{t('Publishing')}</Wrapper>
+      <Wrapper margin="0 0 32px 0">{t('Publishing')}</Wrapper>
```
Comment (himself65) [inline packages/app/src/components/workspace-setting/PublishPage.tsx]:
```suggestion
      <Wrapper marginBottom="32px">{t('Publishing')}</Wrapper>
```

**Review by doodlewind — APPROVED** [review]

File: packages/app/package.json [hunk 8]
```diff
@@ -9,21 +9,22 @@
...
-    "@emotion/styled": "^11.10.4",
+    "@emotion/styled": "^11.10.5",
     "@fontsource/poppins": "^4.5.10",
     "@fontsource/space-mono": "^4.5.10",
-    "@mui/base": "^5.0.0-alpha.87",
-    "@mui/icons-material": "^5.10.9",
-    "@mui/material": "^5.8.6",
+    "@mui/base": "=5.0.0-alpha.101",
+    "@mui/icons-material": "=5.10.9",
+    "@mui/material": "=5.8.6",
```
Comment (himself65) [inline packages/app/package.json:27]: there is a breaking change in 5.11.x

**Comment by himself65** [comment]
gonna merge when @doodlewind create a vercel project

File: packages/app/package.json (hunk 8 above)
Comment (darkskygit) [reply packages/app/package.json:27]: its recommend to submit it as an issue and fix it as soon as possible to avoid version locking

**Review by darkskygit — APPROVED** [review]

**Pull request merged**